benchmarks/Microsoft.AspNetCore.Routing.Performance/Matchers/MinimalMatcher.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+
+namespace Microsoft.AspNetCore.Routing.Matchers
+{
+    internal class MinimalMatcher : Matcher
```

Comment:
This is a 'minimal' implementation that is only here to try and establish the 'minimum' baseline. The idea is that we can cut corners and make this a stable 'theoretical lower bound' for comparison. These kinds of tests work best when you have a baseline implementation, so it seems worthwhile to me.